### DISCUSSION TITLE
Make MODKEY flexible

### DISCUSSION THREAD

**Discussion by mrwsl** [bakkeby/dwm-flexipatch#231] [Ideas]
If one changes https://github.com/bakkeby/dwm-flexipatch/blob/96820b2d51b7e1bca5ccdd9ce289e9fd278aa767/config.def.h#L714
for example to `Mod4Mask`  a lot of the shortcuts become ambiguous.

So the idea would be some sort of case:
Based on the value of the `MODKEY` define an `ALTMODKEY` that has either the value `Mod4Key` or `Mod1Key`.

**Comment by bakkeby** [comment]
Not sure if that makes it more intuitive.

Perhaps it is better to have a more generic search and replace along these lines:
```sh
sed -i 's/Mod4Mask/Mod1Mask/;s/#define MODKEY Mod1Mask/#define MODKEY Mod4Mask/' config.h
```

**Reply by mrwsl** [reply]
I actually did it like this and thought my idea would improve the use a different modkey. But it is just an idea :)
